pkg/xds/generator/proxy_template.go

```diff
@@ -151,30 +153,54 @@ func (_ OutboundProxyGenerator) Generate(ctx xds_context.Context, proxy *model.P
 	resources := make([]*Resource, 0, len(ofaces))
 	names := make(map[string]bool)
 	sourceService := proxy.Dataplane.Spec.GetIdentifyingService()
-	for _, oface := range ofaces {
+	for i, oface := range ofaces {
```

Comment:
Can we try to split body of this for to something like
```
clusterInfo := clusterInfo()
eds := generateEds()
resources = append(resources, eds)
lds := generateLds()
```
if not then at least comment sections so it's easier to follow

Reply:
done (not exactly like in the example, but it should be an improvement over the previous state)

Reply:
Thanks, it is better now!